alibi/explainers/tests/test_simiarlity/test_backends.py

```diff
@@ -41,3 +42,94 @@ def test_backends(random_cls_dataset, linear_models):
     torch_grads = np.sort(torch_grads)
     tf_grads = np.sort(tf_grads)
     np.testing.assert_allclose(torch_grads, tf_grads, rtol=1e-04)
+
+
+@pytest.mark.parametrize('trainable_emd, grads_shape', [(True, (61, )), (False, (21, ))])
+def test_tf_embedding_similarity(trainable_emd, grads_shape):
+    """Test GradSim explainer correctly handles sparsity and non-trainable layers for tensorflow.
```

Comment:
GradSim -> \`GradientSimilarity\`
tensorflow -> `tensorflow`

Although those won't show up in the docs.
If decide to update, please update everywhere

Reply:
I've updated GradSim -> GradientSimilarity but have not added backticks in the tests